decidim-admin/config/locales/ca.yml

```diff
@@ -636,4 +636,4 @@ ca:
   time:
     formats:
       datepicker: "%m/%d/%Y"
-      timepicker: "%d/%m/%Y %H:%M"
\ No newline at end of file
+      timepicker: "%d/%m/%Y %H:%M"
```

Comment:
By the way, we briefly discussed this [here](https://github.com/decidim/decidim/pull/1535#issuecomment-311286760). @mrcasals Did you get a reply from crowdin?